workflows/sesv2_weekly_mailer/SPECIFICATION.md

```diff
@@ -0,0 +1,78 @@
+# SES v2 Coupon Newsletter Workflow Specification
+
+Use the Amazon Simple Email Service (SES) v2 API to manage a subscription list for a weekly newsletter.
+
+## Prepare the Application
+
+1. Create an email identity.
+
+   - Retrieve the verified email address from the `VERIFIED_EMAIL_ADDRESS` environment variable.
```

Comment:
Why are we doing env variables here instead of just prompting the user which we do in the next step? Is this a best practice choice?